fontforgeexe/fontinfo.c

```diff
@@ -196,6 +196,15 @@ static GTextInfo ibmfamily[] = {
     { (unichar_t *) N_("Sy Miscellaneous"), NULL, 0, 0, (void *) 0xc0f, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
     GTEXTINFO_EMPTY
 };
+static GTextInfo stylemap[] = {
```

Comment:
I didn't make this repr so I have nothing particular to say about it. If you want me write something before the stack of arrays, I can do that.